libs/components/core/src/lib/modules/dynamic-component/dynamic-component.service.ts

```diff
@@ -73,9 +67,10 @@ export class SkyDynamicComponentService {
       parent: this.#injector,
     });
 
-    const componentRef = this.#componentFactoryResolver
-      .resolveComponentFactory<T>(componentType)
-      .create(injector);
+    const componentRef = createComponent<T>(componentType, {
```

Comment:
This is neat; haven't seen it before and I think it works well in the context of the dynamic component service. 👍